lib/hammer_cli_foreman_webhooks/webhook_template.rb

```diff
@@ -95,6 +95,24 @@ class DeleteCommand < HammerCLIForeman::DeleteCommand
       build_options
     end
 
+    class ExportCommand < HammerCLIForeman::DownloadCommand
+      command_name "export"
+      action :export
+
+      desc _('Export a webhook template to ERB')
+
+      def default_filename
+        "Webhook Template-#{Time.new.strftime("%Y-%m-%d")}.txt"
+      end
+
+      def msg
```

Comment:
I guess this can wait until we finish https://github.com/theforeman/hammer-cli-foreman/pull/598 :)